test/controllers/VatObligationsControllerSpec.scala

```diff
@@ -30,18 +30,35 @@ class VatObligationsControllerSpec extends SpecBase with MockVatObligationsServi
   val success: VatObligations =
     VatObligations(
       Seq(VatObligation(
-        ObligationIdentification("555555555", "VRN"),
         Seq(
           ObligationDetail("F", "1980-02-03", "1980-04-05", Some("1980-02-02"), "1980-04-08", "17AA"),
           ObligationDetail("F", "1981-02-03", "1981-04-05", Some("1981-02-02"), "1981-04-08", "18AA")
         )
       )
       ))
 
+  val successMultipleObligations: VatObligations =
```

Comment:
Same question here as in the HttpParserSpec - is multiple arrays of obligations a valid scenario?

Reply:
Again. Yes according to the schema.
Code handles the situation where it happens so a valid test.